crates/oxc_linter/src/utils/jest/parse_jest_fn.rs

```diff
@@ -546,6 +546,13 @@ const VALID_JEST_FN_CALL_CHAINS: [[&str; 4]; 51] = [
     ["describe", "skip", "each", ""],
     ["fdescribe", "", "", ""],
     ["fdescribe", "each", "", ""],
+    // vitest
```

Comment:
Since there are too many items in [this file](https://github.com/veritem/eslint-plugin-vitest/blob/9885b9f2b15d316d2d4c2d8f8da410f2aa24c4a1/src/utils/valid-vitest-fn-call-chains.ts#L1), we can add another `VALID_VITEST_FN_CALL_CHAINS` and need to change it to a `set`.
 
 You can use `ctx.frameworks().is_vitest()` to decide which to use.
 
 https://github.com/oxc-project/oxc/blob/7a75e0f8a77006ef32fa0db45a329d961e70bdf3/crates/oxc_linter/src/rules/jest/expect_expect.rs#L140

Reply:
Done.